src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/DeleghePagoPATest.java

```diff
@@ -220,6 +228,17 @@ public void siSceglieOpzioneAccetta() {
         this.leTueDelegheSection.clickOpzioneAccetta();
     }
 
+    @And("Si inserisce il codice delega nel pop-up OTP")
+    public void inserisceCodiceOTPDelega(){
+        this.leTueDelegheSection.waitPopUpLoad();
+        this.leTueDelegheSection.inserireCodiceDelega(codiceVerifica);
```

Comment:
why you use this?? if you create delegation with api exist mandate singleton

Reply:
for this delegation otp code was changing every time, so i used this method to insert every time new otp